.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
     - MAVEN_SKIP_CHECKS_AND_DOCS="-Dair.check.skip-all=true -Dmaven.javadoc.skip=true"
     - MAVEN_FAST_INSTALL="-DskipTests $MAVEN_SKIP_CHECKS_AND_DOCS -B -q -T C1"
   matrix:
-    - MAVEN_CHECKS=true
```

Comment:
change commit message to state that this is change from artifacts being uploaded after `PRODUCT_TESTS` run